cmd/wallet-web/src/components/FlyoutMenu/FlyoutMenu.vue

```diff
@@ -33,47 +28,53 @@
         border border-neutrals-chatelle
         hover:border-neutrals-mountainMist-light
       "
-      @focus="showTooltip = !showTooltip"
+      @focus="showTooltip = false"
     >
-      <div class="flex-none p-2">
-        <!-- TODO: Issue-816 Implement svg color change on hover -->
-        <img
-          id="flyoutMenuId"
-          src="@/assets/img/more-icon.svg"
-          @click="toggleFlyoutMenuList"
-          @mouseover="showTooltip = !showTooltip"
-        />
-      </div>
-      <div v-if="showTooltip" id="tooltip">
-        <tool-tip :tool-tip-label="t('CredentialDetails.toolTipLabel')"></tool-tip>
-      </div>
+      <!-- TODO: Issue-816 Implement svg color change on hover -->
+      <img
+        id="flyoutMenuId"
```

Comment:
You still have this static id

Reply:
Automation test fails , we need this id

Reply:
Please, adjust the automation test in this case. We can't have multiple identical ids on the page.